microsoft-edge/webview2/how-to/winrt-js-conversion.md

```diff
@@ -123,6 +127,31 @@ console.assert(result.intParam2 == 1);
 
 console.assert(result.intParam3 == 2);
 ```
+
+For array type `out` parameters, the array will need to be passed into the method's parameter list when calling from JavaScript. For non-`void` return type, the result array will replace the array passed in for the method call. For `void` return type, the result array will be the result of the method call.
```

Comment:
```suggestion
For array type `out` parameters, the array will need to be passed into the method's parameter list when calling from JavaScript. For a non-`void` return type, the result array will replace the array that's passed in for the method call. For `void` return type, the result array will be the result of the method call.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which - "Include _that_ even if the sentence is clear without it. It helps to clarify the sentence structure."